pkg/component/runtime/command_test.go

```diff
@@ -31,6 +32,9 @@ func TestAddToBucket(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
+			if runtime.GOOS == "windows" {
```

Comment:
https://github.com/elastic/elastic-agent/issues/3290

We've tried to fix these on Windows a few times already, and since there is nothing Windows specific to this test let's just skip it completely on that OS rather than waste more timing trying to account for the variation between CI machines.